### PR TITLE
refactor(v2): clarify versions page

### LINKED ISSUE #3572: Stable unreleased alpha with no release notes
## 📚 Documentation

The [versions](https://v2.docusaurus.io/versions) page shows 2.0.0-alpha.66 as a stable unreleased alpha, which seems like a contradiction. Improved terminology or at least some explanation on the page would be helpful to avoid confusion. I was hoping the [Release Notes](https://github.com/facebook/docusaurus/releases/tag/vcurrent) would clear things up, but the link 404s.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Resolves #3572

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Preview

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/src/pages/versions.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ function Version() {
   const pastVersions = versions.filter(
     (version) => version !== latestVersion && version.name !== 'current',
   );
-
+  const stableVersion = pastVersions.shift();
```
Comment (slorber) [inline website/src/pages/versions.js:23]:
```suggestion
  const stableVersion = pastVersions.shift() ?? latestVersion;
```

**Review by slorber — CHANGES_REQUESTED** [review]
Hi,

LGTM, but as we deploy the bootstrap theme without any version, the pastVersions array is empty (versioning disabled so that netlify build is not too slow due to multiple builds)

We should rather fallback to "latestVersion", or consider in render that stableVersion may be undefined.

File: website/src/pages/versions.js (hunk 1 above)
Comment (slorber) [reply website/src/pages/versions.js:23]: (just an idea, this wn't lead to a good deploy preview versions page for the bootstrap theme, but probably not a big deal anyway)

Comment (lex111) [reply website/src/pages/versions.js:23]: This solution is ok? https://github.com/facebook/docusaurus/pull/3577/commits/84aa3c7d463b6030e338441978f4ff67ee28f140

Comment (slorber) [reply website/src/pages/versions.js:23]: looks even better ;)

**Pull request merged**